### PR TITLE
refactor: Remove unnecessary code not being used

### PR DESCRIPTION
**Pull request by shortdiv**
@zachwise, can you take a look and let me know what you think? I removed external factory function that we weren't using to reduce file size. I'm confident that we can write our own functions or include them back in as the need arises. 

ping @JoeGermuska and @rapoulson; this could use more 👀

### CODE REVIEW

File: src/js/compositions.js [hunk 1]
```diff
@@ -57,13 +49,22 @@ trace = function( msg ) {
...
+    }
+    return el;
+}
+
 KL.Pullquote = (function() {
 
     // DOM ELEMENTS
     this.el = {
-        container: KL.Dom.get("pullquote-container"),
+        container: document.getElementById("pullquote-container"),
```
Comment (JoeGermuska) [inline src/js/compositions.js]: I definitely think this wins for clarity, and there are no cross-browser issues with `getElementById` anymore so big 👍  to this

File: src/js/dom/KL.DomUtil.js [hunk 2]
```diff
@@ -1,157 +0,0 @@
...
-        
-            return w;
-        }).replace(/^\s+/, '');
-    },
-
-    setOpacity: function (el, value) {
-        if (KL.Browser.ie) {
-            el.style.filter = 'alpha(opacity=' + Math.round(value * 100) + ')';
-        } else {
-            el.style.opacity = value;
```
Comment (JoeGermuska) [inline src/js/dom/KL.DomUtil.js:108]: 👍  Opacity has had full support since IE 9 http://caniuse.com/#search=opacity

**Comment by JoeGermuska** [comment]
In general, I'm going to be in favor of less code always. And I found at least one IE specific thing which isn't even relevant anymore, so it's good to shed that kind of stuff.

I don't see any issues here that can't be solved by bringing individual helper methods in if they turn out to be useful.

**Comment by shortdiv** [comment]
I think this PR is also part of a large discussion in terms of making sure we don't deeply couple functions. Right now a lot of the libraries are needed (despite only really needing one function in them) because they are used in multiple other libraries. We need to maintain a separation of concerns so different parts of the code handles specific tasks w/o relying on other pieces of code to work. 

This is also part of a discussion we should have at some point about testing—unit + integration. We don't currently test our code, but I think that we should move toward doing that more. This could help keep us in line about maintaining a separation of concerns when we code. It doesn't necessarily have to be TDD, after the fact regression tests are fine too. Just as long as we have some way of testing stuff and making sure we are writing code for OSS responsibly.

**Comment by rapoulson** [comment]
Manual testing shows that this works and 👍  to making things more lightweight. Agree that it would be great if we had an automated way to tell if we're breaking stuff when things are tightly coupled, but I'm going to go ahead and merge.

**Pull request merged**